micropub.php

```diff
@@ -254,12 +254,12 @@ public static function user_url( $me ) {
 	 */
 	public static function post_handler( $user_id ) {
 		$status = 200;
-		$action = static::$input['action'];
-		$url    = static::$input['url'];
+		$action = isset( static::$input['action'] ) ? static::$input['action'] : 'create';
```

Comment:
we have so many of these, i wonder if we should start using something like the `get()` helper here: http://heap.ch/blog/2015/12/19/php-default-values/#convenience . not necessary for this PR, just an idea. thoughts?

Reply:
I have a similar function I created in most of my plugins called ifset that looks pretty much the same.

Reply:
ah, nice i'm all for adding it here and using it everywhere! (and i like `get` better than `isset`, since it returns a value and not a boolean...but that's a bikeshed. :P)

also, ooh, php 7 added an `??` operator for this! https://wiki.php.net/rfc/isset_ternary . nice to know.

Reply:
WordPress will hopefully get there before I retire.